### ISSUE TITLE
feat: add Localization to FluentPaginator

### ISSUE THREAD

**Issue by damianog** [microsoft/fluentui-blazor#429]
# 🙋 Feature Request

Add string localization to FluentPaginator

## 🤔 Expected Behavior

Static string text should be displayed in local language

## 😯 Current Behavior

The pager display text in plain English

## 💁 Possible Solution

Implement IStringLocalizer

## 🔦 Context

I want to display the paginator text strings according to browser language.
Users ask for text translation.
I considered using a custom component but native is better.

## 💻 Examples

Current string "items", "Page", "of" are hardcoded:

![image](https://github.com/microsoft/fluentui-blazor/assets/2786682/97a86578-368c-4826-b948-b2aa550ea9d8)

Browsing in Italian language I should see:

![image](https://github.com/microsoft/fluentui-blazor/assets/2786682/9fe16e70-c866-4b69-b9f4-2e71d478ccb2)

**Comment by vnbaaij** [comment]
Good suggestion. We would love to get a PR for this (and other languages)

**Comment by damianog** [comment]
Hi @vnbaaij 

What is better for you? A single Resources folder inside the main project with all localized resource files or a separate project?

**Comment by dvoituron** [comment]
It's likely that a single folder in the Core project would be best. To avoid a multitude of small Dlls (at least for the moment).

**Comment by vnbaaij** [comment]
We are thinking to have a resx file per component (to keep it manageable) all in one Resources folder. We are working on this for the next version.

**Comment by dvoituron** [comment]
I come back on my previous comment.

I don't think it's the library's role to manage translations.
It's a process that's far too complex depending on the number of languages to be maintained and on the specificity of each language: singular/plural, feminine/masculine, ...

On the other hand, I think it's preferable to give the developer a way to manage "localizable" content via RenderTemplate or string properties. For example: 'DialogParameters.DismissTitle.

**Comment by vnbaaij** [comment]
This has been resolved by providing `SummaryTemplate` and `PaginationTextTemplate` parameters

**Comment by Tolitech** [comment]
Hello, @vnbaaij .
How about the titles of the buttons Go to first, Go to previous, Go to next, and Go to last could also be passed as parameters?

Currently, I will need to rewrite the logic in the PaginationTextTemplate just to allow putting the button titles in the user's culture.

**Comment by vnbaaij** [comment]
We will address localization in a later version.
